lib/babel.js

```diff
@@ -54,14 +94,22 @@ async function buildFile (sourceFile, targetFile, options = {}) {
 	await fs.writeFile(targetFile, targetCode, "utf-8")
 }
 
-let javascriptExtensions = [".js"]
+let JAVASCRIPT_FILE_EXTENSIONS = [".js"]
```

Comment:
Should this be a `const`?

```suggestion
const JAVASCRIPT_FILE_EXTENSIONS = [".js"]
```